internal/cri/server/sandbox_remove.go

```diff
@@ -113,6 +113,14 @@ func (c *criService) RemovePodSandbox(ctx context.Context, r *runtime.RemovePodS
 
 	sandboxRemoveTimer.WithValues(sandbox.RuntimeHandler).UpdateSince(start)
 
+	ociRuntime, err := c.config.GetSandboxRuntime(sandbox.Config, sandbox.Metadata.RuntimeHandler)
```

Comment:
let's get the ociRuntime earlier in the func eg. before c.generateAndSendContainerEvent ... and while the sandbox is still in the store..